src/components/step-flow/step-flow.component.tsx

```diff
@@ -180,7 +195,14 @@ export const StepFlow = forwardRef<StepFlowHandle, StepFlowProps>(
     );
 
     return (
-      <StyledStepFlow {...rest} {...tagComponent("step-flow", rest)}>
+      <StyledStepFlow
+        role="group"
```

Comment:
This role has been added because I saw an Axe warning of `attribute is not well supported on a div with no valid role attribute` when using the aria props. The role of `group` was what I thought fit this component. However, I am open to suggestions on which is best. A role of `presentation` or `none` triggers the Axe warning.